tests/test_packages/test_skills_integration/test_simple_oracle.py

```diff
@@ -34,8 +34,17 @@
     FUNDED_ETH_PRIVATE_KEY_3,
     MAX_FLAKY_RERUNS_ETH,
     UseGanache,
+    UseLocalFetchNode,
 )
 
+@pytest.mark.integration
+class TestOracleSkillsFetchLedger(AEATestCaseManyFlaky, UseLocalFetchNode):
+    """Test that oracle skills work on the Fetch ledger."""
+
+    @pytest.mark.ledger
+    @pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS_ETH)  # cause possible network issues
+    def test_oracle(self, fetchd):
```

Comment:
This won't run anything atm

Reply:
Just wanted to test the starting of the local node somehow. Since this is working, I'll add an actual test now.